content/blog/jt15s/2020-12-31_hello_from_the_haiku_promotion_team.md

```diff
@@ -0,0 +1,56 @@
++++
+type = "blog"
+title = "Hello from the Haiku Promotion 'Team'!"
+author = "jt15s"
+date = "2020-12-31 21:33:25+11:00"
+tags = ["haiku", "software"]
++++
+Hi there, I'm jt15s, a new Haiku community member! I've been following the project for a few years now since I stumbled upon an article about Haiku. 
+
+If you read the title (which I'm assuming you did), you’re probably going, "wait, we have a promotion team now?" Well, yes, now we do. As an informal “team” of two (currently), we are super excited to share our plans to help promote Haiku and make more people aware of it.
+
+## Who's in it?
+Me, as well as scott_puopolo18 as of now. We are also working with Haikunauts and BeSly, two Haiku-related resources. However, anyone who wishes to help is more than welcome to! Reach out to me (jt15s) [on the forums](https://discuss.haiku-os.org) if you are interested. If you don't have a forum account yet, create one - a forum account is free! We are also looking for the following informal roles:
+* Video Editor - it would be good to have someone produce video content for Haiku. Videos wouldn’t need to be too complex: just something that looks simple yet professional and something less than 10 minutes long.
+* Presenter - we need someone who is a confident speaker and who can speak in front of a microphone/camera (preferably with a good mic/camera setup too) to be the voice (or maybe even face) of the Haiku videos we make, to give them a more “human touch”.
+* Graphic Designer - it would be nice if we had someone who was solely responsible for producing graphics for Haiku promotion. Graphics should be uploaded to both the website and Haiku Artwork repository using Github, but I’m more than happy to help teach people how to do that.
+* Live Stream Technician - we are considering doing live streams, but we will need someone with technical knowledge of live-streaming who can help facilitate this.
+* Events Coordinator - it would be good to have someone search for events which Haiku could take part in (i.e. FOSS conferences) and help organise Haiku’s participation in these events.
+
+## Our promotional strategy
+As the promotion team, we want to make sure people know about Haiku. One of the biggest problems facing the project is that we are more obscure than other FOSS projects. This means it's harder to get new contributors, whether this is developers, documentation writers or just volunteers in general. Hence we aim to attract more attention towards Haiku, and in turn, new users and volunteers. 
```

Comment:
I feel like Haiku is in the more obscure category: I feel if I asked any random stranger on the street they would have not heard of Haiku - maybe one would remember it from its earlier days but I wouldn't count on it. The chances of a younger person having heard of Haiku are even less - most of them might not even know about BeOS. Hence why we need more awareness and attention, such as that article on Lilliputing that mentioned how Medo fits on a floppy disk (of all the headlines they could have made, they used that one...).

Reply:
Since we cannot take an accurate survey of just how obscure Haiku is, while painting ourselves as a bit of an underdog is sensible from a PR point of view, you could just try making it less strong, e.g.:

```suggestion
As the promotion team, we want to make sure people know about Haiku. One problem facing the project is that we may be more obscure than other FOSS projects. This would mean it's harder to get new contributors, whether this is developers, documentation writers or just volunteers in general. Hence we aim to attract more attention towards Haiku, and in turn, new users and volunteers. 
```